operations/model.go

```diff
@@ -155,9 +159,50 @@ func (s *ClientSettings) setupRestCommunicator(ctx context.Context, printMessage
 	if printMessages {
 		printUserMessages(ctx, c, !s.AutoUpgradeCLI)
 	}
+
+	if s.shouldGenerateJWT(ctx, c) {
+		grip.Info("We will attempt to generate a JWT token, to opt out of this, set 'do_not_run_kanopy_oidc' to true in your config file")
+		if s.JWT, err = RunKanopyOIDCLogin(); err != nil {
+			grip.Warningf("Failed to get JWT token: %s", err)
```

Comment:
nit: is this an error or a warning? I'd generally use a warning only if it falls back to some other behavior. In this case, if it falls back to the API key auth, then a warning is reasonable as long as it says what it's falling back to trying. If this is a stop-on-error kind of scenario though, it seems more like an error would be clearer to the user.

Reply:
I added a note to DEVPROD-17404